src/App.svelte

```diff
@@ -121,678 +182,143 @@
-            </p>
-            <div
-              class="hidden lg:inline-flex md:inline-flex flex flex-inline"
-            >
-              <div class="flex flex-inline">
-                <a target="_blank" href="https://github.com/eddiejaoude">
-                  <i
-                    class="fa fa-github-alt pr-1 pt-6"
-                    style="
-                      font-size: 18px;
-                      color: rgb(109, 71, 199);
-                      font-style: normal;
-                    "
-                  ></i>
-                  <p
-                    class="inline pr-5 pt-5 text-purple-500 text-lg font-medium"
-                  >
-                    GitHub
-                  </p>
-                </a>
-              </div>
-              <div class="flex flex-inline">
-                <a
-                  target="_blank"
-                  href="http://youtube.com/eddiejaoude?sub_confirmation=1"
+            <div class="flex flex-inline">
+              <a target="_blank" href={data.github}>
+                <i
+                  class="fa fa-github-alt pr-1 pt-6"
+                  style="
+                    font-size: 18px;
+                    color: rgb(109, 71, 199);
+                    font-style: normal;
+                  "
+                ></i>
+                <p
+                  class="inline pr-5 pt-5 text-purple-500 text-lg font-medium"
                 >
-                  <i
-                    class="fa fa-link pr-1 pt-6"
-                    style="
-                      font-size: 18px;
-                      color: rgb(109, 71, 199);
-                      font-style: normal;
-                    "
-                  ></i>
-                  <p
-                    class="inline pr-5 pt-5 text-purple-500 text-lg font-medium"
-                  >
-                    Website
-                  </p>
-                </a>
-              </div>
+                  GitHub
+                </p>
+              </a>
             </div>
-            <!-- for small screens -->
-            <div
-              class="flex-inline lg:hidden sm:flex-inline md:hidden flex flex-inline"
-            >
-              <div>
-                <a target="_blank" href="https://github.com/eddiejaoude">
-                  <i
-                    class="fa fa-github-alt pt-5 pr-5"
-                    style="font-size: 20px; color: rgb(109, 71, 199)"
-                  ></i>
-                </a>
-              </div>
-              <div>
-                <a
-                  target="_blank"
-                  href="http://youtube.com/eddiejaoude?sub_confirmation=1"
+            <div class="flex flex-inline">
+              <a target="_blank" href={data.website}>
+                <i
+                  class="fa fa-link pr-1 pt-6 text-purple-500"
+                  style="
+                    font-size: 18px;
+                    color: rgb(109, 71, 199);
+                    font-style: normal;
+                  "
```

Comment:
Since we have the `text-purple-500` class here, we don't need the other color style 👍. Can you make this change?  
```suggestion
                  class="fa fa-link pr-1 pt-6 text-purple-500"
                  style="
                    font-size: 18px;
                    font-style: normal;
                  "
```

Reply:
Will do 😉